src/main/java/Duke.java

```diff
@@ -1,10 +1,89 @@
+import java.util.Scanner;
 public class Duke {
+    static final int MAX_SIZE = 100;
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printsGreeting();
+        mimicMessage();
+
+    }
+
+    private static void printsGreeting() {
+        String greetingMessage = "Hello! I'm ThawBot!\nWhat can I do for you?\n";
+        System.out.println(greetingMessage);
+    }
+
+    private static void printGoodByeMessage() {
+        String goodbyeMessage = "Bye. Hope to see you again soon!";
+        System.out.println(goodbyeMessage);
+    }
+
+    private static void mimicMessage() {
+        Scanner input = new Scanner(System.in);
+        Task[] list = new Task[MAX_SIZE];
+        int currentIteration = 0;
+        boolean canExit = false;
+
+        while (!canExit) {
+            String usersInput = input.nextLine();
+
+            if (usersInput.equals("bye")) {
+                canExit = true;
+                printGoodByeMessage();
+            } else if (usersInput.equals("list")) {
+                printList(currentIteration, list);
```

Comment:
Could use a few more comments here and there especially for the longer methods, but otherwise the code is very clear and easy to understand.